package.json

```
@@ -34,8 +34,9 @@
     "pretest:integration": "concurrently \"npm run build:prod-sample\" \"docker-compose -f ./prod-sample/docker-compose.yml pull nginx selenium-chrome\" --kill-others-on-fail -n build-prod-sample,build-dependency-images",
     "test:integration": "JEST_TEST_REPORT_PATH=./test-results/integration-test-report.html jest integration --config jest.integration.config.js --forceExit",
     "test:danger": "if-env NODE_ENV=production && danger ci || danger local",
-    "test": "npm run test:lint && npm run test:lockfile && npm run test:unit && npm run test:integration && npm run test:danger && npm run test:git-history",
-    "test:git-history": "commitlint --from origin/main --to HEAD",
+    "test": "npm run test:lint && npm run test:lockfile && npm run test:unit && npm run test:integration && npm run test:danger && npm run test:git-history-main",
+    "test:git-history": "commitlint --to HEAD",
+    "test:git-history-main": "commitlint --from origin/main --to HEAD",
```

not sure this would be better showing that in the script

```suggestion
    "test:git-history": "commitlint --to HEAD",
    "test:git-history-main": "npm run test:git-history -- --from origin/main",
```